packages/components/app/styles/components/avatar.scss

```diff
@@ -0,0 +1,24 @@
+//
+// AVATAR
+//
+// properties within each class are sorted alphabetically
+//
+
+.hds-avatar {
+  align-items: center;
+  box-sizing: border-box;
+  display: inline-flex;
+  height: 32px;
+  justify-content: center;
+  width: 32px;
+}
+
+.hds-avatar img {
+  border-radius: 2px;
+  height: 32px;
+  width: 32px;
+}
+
+.hds-avatar svg {
+  display: inline-block;
+}
```

Comment:
suggestion: You could make use of Sass's nested syntax to avoid repeating the parent selector.